tools/scripts/generate_constant_assets.py

```diff
@@ -1,18 +1,27 @@
 from pathlib import Path
 from typing import Dict
 
+from gevent import monkey  # isort:skip # noqa
+monkey.patch_all()  # isort:skip # noqa
+
 from rotkehlchen.config import default_data_directory
 from rotkehlchen.constants.resolver import ChainID, strethaddress_to_identifier
 from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.logging import TRACE, add_logging_level
 from rotkehlchen.utils.misc import timestamp_to_date, ts_now
 
 
+add_logging_level('TRACE', TRACE)
+
 class ContextManager():
     """Manages the parsing context of the assets template"""
 
     def __init__(self) -> None:
         self.id_to_variable: Dict[str, str] = {}
-        self.globaldb = GlobalDBHandler(default_data_directory())
+        self.globaldb = GlobalDBHandler(
+            data_dir=default_data_directory(),
+            sql_vm_instructions_cb=0,
```

Comment:
There's a `DEFAULT_SQL_VM_INSTRUCTIONS_CB` constant. Maybe you should use that instead

Reply:
0 here means no switch. That setting defaults to 5000 and is used because we spawn a lot of tasks and we have to make sure every task leaves time for the rest. In this case there is no other task to switch to so setting it to 0 I believe is the best value here

Reply:
Ohh Ohh alright then.

LGTM